### PR TITLE
Review Sample Relationship

### PR DESCRIPTION
**Pull request by KellyVance**
Sample relationship vocabulary as required by SRA
Only to Level One. 
Follows basic SOSA  sample relationships
has, isOf, hasOriginal.

### CODE REVIEW

File: vocabularies/sample-relationship.ttl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+smprl:is-sample-of a skos:Concept ;
+    skos:definition "Relation from aï¿½Sampleï¿½to theï¿½FeatureOfInterest, Site, or parent Sampleï¿½that it is intended to be representative of."@en ;
+    skos:inScheme smprl:conceptScheme ;
+    skos:notation "isSampleOf" ;
+    skos:prefLabel "Is Sample Of"@en ;
+    skos:topConceptOf smprl:conceptScheme .
+    
+<http://linked.data.gov.au/org/gsq>
+    a sdo:Organization ;
```
Comment (LukeHauck) [inline vocabularies/sample-relationship.ttl]: Turtle validator is giving me "_Undefined prefix "sdo:" on line 45._"

File: vocabularies/sample-relationship.ttl [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    skos:prefLabel "Sample Relationship"@en .
+smprl:has-originalsample a skos:Concept ;
+    skos:definition "The link to the original sample that is related to the context sample through a chain of isSampleOf relations"@en ;
+    skos:inScheme smprl:conceptScheme ;
+    skos:notation "hasOriginalSample" ;
+    skos:prefLabel "Has OriginalSample"@en ;
+    skos:topConceptOf smprl:conceptScheme .
+
+smprl:has-sample a skos:Concept ;
+    skos:definition "Relation between aï¿½FeatureOfInterest, Site, or parent Sampleï¿½and theï¿½Sampleï¿½used to represent it."@en ;
```
Comment (LukeHauck) [inline vocabularies/sample-relationship.ttl]: Weird characters in definition.

File: vocabularies/sample-relationship.ttl [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    skos:inScheme smprl:conceptScheme ;
+    skos:notation "hasSample" ;
+    skos:prefLabel "Has Sample"@en ;
+    skos:topConceptOf smprl:conceptScheme .
+
+smprl:is-sample-of a skos:Concept ;
+    skos:definition "Relation from aï¿½Sampleï¿½to theï¿½FeatureOfInterest, Site, or parent Sampleï¿½that it is intended to be representative of."@en ;
```
Comment (LukeHauck) [inline vocabularies/sample-relationship.ttl]: Weird characters in definition.

File: vocabularies/sample-relationship.ttl (hunk 2 above)
Comment (KellyVance) [reply vocabularies/sample-relationship.ttl]: removed

File: vocabularies/sample-relationship.ttl (hunk 3 above)
Comment (KellyVance) [reply vocabularies/sample-relationship.ttl]: removed

File: vocabularies/sample-relationship.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/sample-relationship.ttl]: I get the same. But it's a copy and paste of a Nick addition for defining the organisation.

**Comment by KellyVance** [comment]
This definitely needs expansion. But for the top level I think its a logical place to pause.

I see that fairly quickly we might need to add things under isSampleOf like isSplitOf, isDuplicateOf, isExtractOf etc.

**Comment by DavidCrosswellGSQ** [comment]
> This definitely needs expansion. But for the top level I think its a logical place to pause.
> 
> I see that fairly quickly we might need to add things under isSampleOf like isSplitOf, isDuplicateOf, isExtractOf etc.

Yes - agree with place to pause.

**Pull request merged**